pkg/operator/sync.go

```diff
@@ -183,7 +183,11 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 			corev1.ResourceCPU:    resource.MustParse("10m"),
 		},
 	}
-	args := []string{"--logtostderr=true", "--v=3"}
+	args := []string{
+		"--logtostderr=true",
+		"--v=3",
+		"--namespace=openshift-machine-api",
```

Comment:
This needs to be parametrized. `ci/jenkins/integration` expects all machines to live under `kube-system`, not `openshift-machine-api`.